README.md

```diff
@@ -0,0 +1,17 @@
+# Shadow Tool Spring Boot Starter
+
+## Introduction
+For use with Spring Boot applications. See [Shadow Tool](https://github.com/rabobank/shadow-tool) for more information.
+
+## Installation
+
+### Maven
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.github.rabobank.shadow_tool/shadow-tool-spring-boot-starter/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.github.rabobank.shadow_tool/shadow-tool-spring-boot-starter)
+
+```xml
+<dependency>
+    <groupId>io.github.rabobank.shadow_tool</groupId>
+    <artifactId>shadow-tool-spring-boot-starter</artifactId>
+    <version>[check maven central for latest version]</version>
+</dependency>
+```
```

Comment:
Let's extend it with how to use it too. What kind of properties you can set and how to configure those Shadow Flow beans would certainly be something for this starter, not for the tool itself.